specification/standbypool/StandbyPool.Management/client.tsp

```diff
@@ -61,3 +61,8 @@ using Microsoft.StandbyPool;
   "javascript"
 );
 @@clientName(Microsoft.StandbyPool, "StandbyPoolMgmtClient", "python");
+@@clientName(Microsoft.StandbyPool, "StandbyPoolManagementClient", "java");
```

Comment:
@XiaofeiCao Opinion whether we break here? Current GA is `StandbyPoolCient`.

Reply:
Yeah, for consistency, I think we have to.